Keas.Mvc/ClientApp/components/Equipment/EquipmentEditValues.tsx

```diff
@@ -56,7 +57,7 @@ export default class EquipmentEditValues extends React.Component<IProps, {}> {
                     <input type="text"
                         className="form-control"
                         disabled={true}
-                        value={this.props.selectedEquipment.assignment.expiresAt.toString()}
+                        value={moment(this.props.selectedEquipment.assignment.expiresAt).format("MM/DD/YYYY").toString()}
```

Comment:
@jcubbage how about instead you make a quick util (kinda like the permissions util) and just have it export a method like formatExpiration() that takes a date.  It can handle the null checks, importing moment and defining the format.  Should make the code cleaner plus if we want to change the format slightly later we just do it in one place